app/src/main/java/com/craxiom/networksurveyplus/mqtt/QcdmMqttConnection.java

```diff
@@ -46,7 +52,12 @@ public void onQcdmMessage(QcdmMessage qcdmMessage)
                 case QcdmConstants.LOG_LTE_RRC_OTA_MSG_LOG_C:
                     convertAndPublishLteRrcMessage(qcdmMessage);
                     break;
-
+                case QcdmConstants.UMTS_NAS_OTA:
```

Comment:
There is also the UMTS_NAS_OTA_DSDS case that we want to handle for MQTT. I already set it up for pcap logging so you should be able to handle it easily with MQTT.